lib/log_snag/client.rb

```diff
@@ -19,7 +19,7 @@ def log(project:, channel:, event:, **params)
 
     def connection
       @connection ||= Faraday.new(BASE_URL) do |conn|
-        conn.request :authorization, :basic, token
+        conn.headers["Authorization"] = "Bearer #{token}"
```

Comment:
Faraday has authorization for Bearer tokens built in:

```
conn.request :authorization, :Bearer, token
```

Reply:
Awesome! Didn't work with faraday much yet, thanks for the tip + implemented!